src/tests/class_tests/openms/source/MSSpectrum_test.cpp

```diff
@@ -46,6 +46,11 @@
 using namespace OpenMS;
 using namespace std;
 
+static_assert(OpenMS::Test::fulfills_rule_of_5<MSSpectrum>(), "Must fulfill rule of 5");
+static_assert(OpenMS::Test::fulfills_rule_of_6<MSSpectrum>(), "Must fulfill rule of 6");
+static_assert(OpenMS::Test::fulfills_fast_vector<MSSpectrum>(), "Must have fast vector semantics");
+static_assert(std::is_nothrow_move_constructible<MSSpectrum>::value, "Must have nothrow move constructible");
```

Comment:
@cbielow @timosachsenberg what do you think of these? Here we can use `constexpr` functions to test these different properties